add.php

```diff
@@ -0,0 +1,38 @@
+<?php
+require './functions.php';
+$db = connectToDatabase();
+
+
+
+if (isset($_POST['name'])) {
+    $name = cleanedInput($_POST['name']);
+    $status = cleanedInput($_POST['status']);
+    $weight = cleanedInput($_POST['weight']);
+    $threat = cleanedInput($_POST['threat']);
+    $diet = cleanedInput($_POST['diet']);
+    $distribution = cleanedInput($_POST['distribution']);
+    $image = cleanedInput($_POST['image']);
```

Comment:
Suggest adding some extra basic validation to these fields.

For example, a check to make sure the $name field is no longer than 20 characters (which is the limit your database can hold).

What you could do is then redirect the user back to index.php using `header()` and display an error message. One way of doing that is to add some GET data into the URL like so:

`header("Location: index.php?error=Invalid name")`

And you could then display that error message on index.php by accessing it in `$_GET` like you would normal form data. 

Let me know if this doesn't make any sense!